roles/disko.nix

```diff
@@ -0,0 +1,49 @@
+{ disks ? [ ], ... }:
+let
+  content = {
+    format = "gpt";
+    type = "table";
+    partitions = [ ];
+  };
+in
+{
+  disk = {
+    disk0 = {
```

Comment:
I was wondering if we should go for a mirror raid since we wear out the NVME quite a bit with write cycles and we don't really need that high of an uptime.

Reply:
Did you mean striping instead of mirroring?

We probably want to have a swap partition as well.

Reply:
Ah. Yes. I mean striping instead of mirroring.